src/js/components/StarRating/StarRating.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { Star, StarOutline } from 'grommet-icons';
+import { RadioButtonGroup } from '../RadioButtonGroup';
+
+export const StarRating = ({
+  fillColor,
+  name,
+  label,
+  scale = 5,
+  value,
+  onChange,
+  outlineColor,
```

Comment:
instead of having `fillColor` and `outlineColor`, it'd be nice to just have a single `color` prop. Where color could be:
- a string (applies to both fill and outline)
- object: { fill: string, outline: string }

Reply:
good idea

Reply:
Ill also do the same for thumbs

Reply:
done